common/sgx/tcbinfo.h

```diff
@@ -15,6 +15,17 @@ OE_EXTERNC_BEGIN
 
 #define OE_TCB_LEVEL_STATUS_UNKNOWN (0)
 
+#define OE_TCB_STATUS_SGX_UP_TO_DATE "UpToDate"
```

Comment:
Please somehow link these definitions to `OE_CLAIM_TCB_STATUS` so people can find possible tcb status in Doxygen.